internal/fs/interface.go

```diff
@@ -1,9 +1,8 @@
 package fs
 
 import (
-	"io"
-
 	"github.com/restic/restic/internal/restic"
+	"io"
```

Comment:
Imports in restic are typically grouped such that the standard library is listed first and the all other dependencies. Please keep that pattern.